### PR TITLE
Fix: Import export from kit library [ED-6684]

### PR DESCRIPTION
**Pull request by Nevoss**

### CODE REVIEW

File: core/app/modules/import-export/module.php [hunk 1]
```diff
@@ -104,7 +104,12 @@ public function get_summary_titles() {
 	private function import_stage_1() {
 		// PHPCS - Already validated in caller function.
 		if ( ! empty( $_POST['e_import_file'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Missing
-			$file_url = $_POST['e_import_file']; // phpcs:ignore WordPress.Security.NonceVerification.Missing
+			if ( ! wp_verify_nonce( $_POST['e_kit_library_nonce'], 'kit-library-import' ) ) {
+				throw new \Error( __( 'Cannot verify import file url', 'elementor' ) );
```
Comment (rodolphebertozzo) [inline core/app/modules/import-export/module.php:108]:
```suggestion
				throw new \Error( esc_html__( 'Cannot verify import file url', 'elementor' ) );
```

**Review by rodolphebertozzo — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/module.php (hunk 1 above)
Comment (Nevoss) [reply core/app/modules/import-export/module.php:108]: Thanks

File: core/app/modules/import-export/module.php [hunk 2]
```diff
@@ -168,7 +176,7 @@ private function import_stage_2( $settings_directory ) {
 		return $result;
 	}
 
-	private function on_admin_init() {
+	public function on_admin_init() {
```
Comment (matipojo) [inline core/app/modules/import-export/module.php:179]: Why

**Review by matipojo — CHANGES_REQUESTED** [review]

File: core/app/modules/import-export/module.php (hunk 2 above)
Comment (Nevoss) [reply core/app/modules/import-export/module.php:179]: mistake

**Pull request merged**